### PR TITLE
Fixed core, waterml, and config imports

### PR DESCRIPTION
**Pull request by lsetiawan**
This will hopefully fix the import core failures during travis ci build.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Thanks for addressing these imports @lsetiawan! There are probably more that needs to be made either absolute, like that, or relative inside the package. I prefer the former BTW.

**Comment by lsetiawan** [comment]
Darn it stopped at config still... looking for a fix.. don't merge yet

File: wof/core_1_0.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 import logging
 
-import core
-import WaterML
+import wof.core as core
```
Comment (ocefpaf) [inline wof/core_1_0.py]: It is more common to use `from wof import core` but I don't have a preference. Feel free to use whatever you think it is best.

Comment (lsetiawan) [inline wof/core_1_0.py]: Ah, I think your way is cleaner! :)

**Comment by ocefpaf** [comment]
> Darn it stopped at config still... looking for a fix.. don't merge yet

Do you mean `Travis-CI`?

I was planning to review all the imports today but I don't want to duplicate the effort if you are tackling this. (I also want to avoid the need for rebasing for both of us  :smile:)

**Comment by lsetiawan** [comment]
> Do you mean Travis-CI?

Yea.

> I was planning to review all the imports today but I don't want to duplicate the effort if you are tackling this. 

Either way, just trying to help you out 😄 Let me know what I can get done.

**Comment by ocefpaf** [comment]
> Either way, just trying to help you out 😄 Let me know what I can get done.

Please do :smile: 

Let's just coordinate the effort. Ping me once you need a review/merge on this one, and let me know if you want to catch further absolute import failures.

I will work on something else and get back to the imports when I hear from you.

Thanks!

**Comment by lsetiawan** [comment]
Yea.. Looks like that would work. Thanks! Changing now.

**Comment by lsetiawan** [comment]
@ocefpaf What do you think? Should I merge this one?

**Comment by ocefpaf** [comment]
> @ocefpaf What do you think? Should I merge this one?

Yep. LGTM.

**Pull request merged**